### ISSUE TITLE
Getting 401 error

### ISSUE THREAD

**Issue by ParthVora777** [atymic/twitter#178]
Hello,

I have this simple code in my routes.php file:

```
Route::get('test', function() {
    $img = $_SERVER['DOCUMENT_ROOT'] . '/img/share.png';
    $uploaded_media = \Twitter::uploadMedia(['media' => File::get($img)]);
    return \Twitter::postTweet(['status' => 'Laravel is beautiful', 'media_ids' => $uploaded_media->media_id_string]);
});
```

but visiting this route gives me 401 error. It is not even redirecting me to the Twitter application accept page. What am I doing wrong here?

I have placed all the keys and tokens correctly into the config files and I'm using Laravel 4.2 version.

Thanks.

**Comment by Gazoo101** [comment]
Have you ensured that your current token has Read **and** Write access?

I was also getting 401 errors, and while I changed Access Level from Read, to Read **and** Write, I also needed to re-issue the token. Something I feel the website really should highlight every time the general permissions is changed.

In other words, it's possible that you've changed the access level but are still using a token with old privileges. :E
